packages/vuetify/src/components/VSelectionControlGroup/VSelectionControlGroup.tsx

```diff
@@ -3,47 +3,65 @@ import './VSelectionControlGroup.sass'
 
 // Composables
 import { IconValue } from '@/composables/icons'
+import { makeDensityProps } from '@/composables/density'
+import { makeThemeProps } from '@/composables/theme'
+import { provideDefaults } from '@/composables/defaults'
 import { useProxiedModel } from '@/composables/proxiedModel'
 
 // Utilities
 import { computed, provide, toRef } from 'vue'
-import { defineComponent, getUid, useRender } from '@/util'
+import { deepEqual, defineComponent, getUid, propsFactory, useRender } from '@/util'
 
 // Types
 import type { InjectionKey, PropType, Ref } from 'vue'
 
 export interface VSelectionGroupContext {
-  disabled: Ref<boolean>
-  inline: Ref<boolean>
-  name: Ref<string | undefined>
   modelValue: Ref<any>
-  multiple: Ref<boolean>
-  trueIcon: Ref<IconValue | undefined>
-  falseIcon: Ref<IconValue | undefined>
-  readonly: Ref<boolean>
-  type: Ref<string | undefined>
 }
 
 export const VSelectionControlGroupSymbol: InjectionKey<VSelectionGroupContext> = Symbol.for('vuetify:selection-control-group')
 
+export const makeSelectionControlProps = propsFactory({
+  color: String,
+  disabled: Boolean,
+  defaultsTarget: {
+    type: String,
+    default: 'VSelectionControl',
+  },
+  error: Boolean,
+  id: String,
+  inline: Boolean,
+  label: String,
+  falseIcon: IconValue,
+  trueIcon: IconValue,
+  ripple: {
+    type: Boolean,
+    default: true,
+  },
+  multiple: {
+    type: Boolean as PropType<boolean | null>,
+    default: null,
+  },
+  name: String,
+  readonly: Boolean,
+  trueValue: null,
+  falseValue: null,
+  modelValue: null,
+  type: String,
+  value: null,
+  valueComparator: {
+    type: Function as PropType<typeof deepEqual>,
+    default: deepEqual,
+  },
+
+  ...makeThemeProps(),
+  ...makeDensityProps(),
+}, 'VSelectionControl')
+
 export const VSelectionControlGroup = defineComponent({
   name: 'VSelectionControlGroup',
 
-  props: {
-    disabled: Boolean,
-    id: String,
-    inline: Boolean,
-    name: String,
-    falseIcon: IconValue,
-    trueIcon: IconValue,
-    multiple: {
-      type: Boolean as PropType<boolean | null>,
-      default: null,
-    },
-    readonly: Boolean,
-    type: String,
-    modelValue: null,
-  },
+  props: makeSelectionControlProps(),
```

Comment:
Some of these don't make sense for the group root, and aren't being used anyway (value, trueValue, falseValue, label)